### PR TITLE
added new branch

### PR DESCRIPTION
**Pull request by chye0001**
review ny kode i ny branch plz

### CODE REVIEW

**Comment by SeveriusRasmussen** [comment]
I don't care about yout project hahaha

File: src/main/java/Main.java [hunk 1]
```diff
@@ -2,6 +2,9 @@ public class Main {
     public static void main(String[] args) {
 
         System.out.println("<Hello WOrld");
+
+
```
Comment (SeveriusRasmussen) [inline src/main/java/Main.java]: Looks great! Now you can go out and do your job and become anomymous.

**Review by LasseThomsenSkole — CHANGES_REQUESTED** [review]
## forslag til ændring
```
System.out.println("Old Branch");

**Review by SeveriusRasmussen — COMMENTED** [review]
Forslag til ændring:
```
System.out.println("Hello kammerater!");
```

**Review by LasseThomsenSkole — APPROVED** [review]

**Pull request merged**